lib/aha-services/networking.rb

```diff
@@ -223,7 +223,11 @@ def check_ssl
   end
 
   def reportable_http_env(env, time)
-    "#{env[:method].to_s.upcase} #{env[:url]} -- (#{"%.02fs" % [Time.now - time]}) #{env[:status]} #{env[:body]} #{env[:response_headers].inspect}"
+    if env[:response_headers]["content-type"] =~ /^text\// || env[:response_headers]["content-type"].starts_with?("application/json")
```

Comment:
We needed to change our logging, because http-get an attachment would pollute the logs and expose customer data.